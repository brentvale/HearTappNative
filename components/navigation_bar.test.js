import 'react-native';
import React from 'react';
import NavigationBar from './navigation_bar';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
	const tree = renderer.create(
		<NavigationBar />
	).toJSON();
	expect(tree).toMatchSnapshot();
})