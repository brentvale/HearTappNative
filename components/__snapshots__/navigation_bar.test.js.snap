// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "backgroundColor": "#fff",
      "flexDirection": "row",
      "height": 75,
      "justifyContent": "center",
      "marginTop": 25,
    }
  }
>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits="button"
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "red",
        "borderRightColor": "white",
        "borderRightWidth": 2,
        "height": 50,
        "opacity": 1,
        "overflow": "hidden",
        "padding": 10,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#007aff",
            "fontSize": 17,
            "fontWeight": "500",
            "textAlign": "center",
          },
          null,
          Object {
            "color": "white",
            "fontSize": 20,
            "width": 200,
          },
          null,
        ]
      }
    >
      Tap Heart Rate
    </Text>
  </View>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits="button"
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "red",
        "height": 50,
        "opacity": 1,
        "overflow": "hidden",
        "padding": 10,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#007aff",
            "fontSize": 17,
            "fontWeight": "500",
            "textAlign": "center",
          },
          null,
          Object {
            "color": "white",
            "fontSize": 20,
            "width": 200,
          },
          null,
        ]
      }
    >
      Learn How
    </Text>
  </View>
</View>
`;
